EIPS/eip-725.md

```diff
@@ -1,249 +1,353 @@
 ---
 eip: 725
-title: General key-value store and execution standard
+title: General data key/value store and execution
+description: An interface for a smart contract based account with attachable data key/value store
 author: Fabian Vogelsteller (@frozeman), Tyler Yasaka (@tyleryasaka)
 discussions-to: https://github.com/ethereum/EIPs/issues/725
```

Comment:
Would you mind upgrading this to a thread on Ethereum Magicians?

Reply:
@Pandapip1 Done 👍